src/bindgen.rs

```diff
@@ -28,49 +28,62 @@ pub fn wasm_bindgen_build(
 
     let out_dir = out_dir.to_str().unwrap();
 
-    let wasm_path = data
-        .target_directory()
-        .join("wasm32-unknown-unknown")
-        .join(release_or_debug)
-        .join(data.crate_name())
-        .with_extension("wasm");
-
-    let dts_arg = if disable_dts {
-        "--no-typescript"
-    } else {
-        "--typescript"
-    };
-    let bindgen_path = bindgen.binary("wasm-bindgen")?;
+    for target_name in data.targets() {
```

Comment:
I think this PR could be a bit more simplified if we remove handling multiple crates here and leave that to #732.

Reply:
I think this is distinct from #732; if a crate has multiple binaries, a single `cargo build` will create multiple `.wasm` files for it, and the same invocation of `wasm-pack` should process all of them. #732 appears to be intended to allow repeated invocations of `wasm-pack` to build on top of each other; the example given uses two different feature sets, which is orthogonal to what this is doing.

Reply:
> #732 appears to be intended to allow repeated invocations of `wasm-pack` to build on top of each other

I didn't realize that 😮! My intention was to slim down the PR to help push this over the finishing line, because I am assuming that the bandwidth of the wasm team is pretty strained.
It would be best if somebody from the wasm team says what he thinks about this of course.